.env

```diff
@@ -9,3 +9,8 @@ MAILGUN_API=fhlakjshdf8932pidkjshfasdjfy89234ks83pikanzkjahslka
 JWT_PRIVATE_KEY_LOCATION=config/keys/jwtRS256.key
 JWT_PUBLIC_KEY_LOCATION=config/keys/jwtRS256.key.pub.pem
 JWT_SECRET_KEY=123
+
+JWT_ISSUER=http://brokencrystals/keycloak/auth/realms/master
+JWT_INTROSPECT_ENDPOINT=http://brokencrystals/keycloak/auth/realms/master/protocol/openid-connect/token/introspect
```

Comment:
You should fetch config from http://localhost:8080/auth/realms/master/.well-known/openid-configuration and read a token endpoint from it

Reply:
just for test